bfh/__init__.py

```diff
@@ -252,17 +252,17 @@ class DogToAnimal(Mapping):
             legs = Const(4)
             noise = Const('woof!')
 
-    The action happens when you get an instance of your mapping:
+    The action happens when you get an instance of your mapping::
 
         dog_to_animal_map = DogToAnimal()
 
-    Then apply it to an object. You get a Schema instance:
+    Then apply it to an object. You get a Schema instance::
```

Comment:
That's an rst literal block http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#literal-blocks